exporters/elasticsearch/include/opentelemetry/exporters/elasticsearch/es_log_recordable.h

```diff
@@ -67,6 +70,39 @@ class ElasticSearchRecordable final : public sdk::logs::Recordable
     }
   }
 
+  void WriteKeyValue(nostd::string_view key,
+                     const opentelemetry::sdk::common::OwnedAttributeValue &value,
+                     std::string name)
+  {
+    namespace common = opentelemetry::sdk::common;
+    switch (value.index())
+    {
+      case common::kTypeBool:
+        json_[name][key.data()] = opentelemetry::nostd::get<bool>(value) ? true : false;
+        return;
+      case common::kTypeInt:
+        json_[name][key.data()] = opentelemetry::nostd::get<int>(value);
+        return;
+      case common::kTypeInt64:
+        json_[name][key.data()] = opentelemetry::nostd::get<int64_t>(value);
+        return;
+      case common::kTypeUInt:
+        json_[name][key.data()] = opentelemetry::nostd::get<unsigned int>(value);
+        return;
+      case common::kTypeUInt64:
+        json_[name][key.data()] = opentelemetry::nostd::get<uint64_t>(value);
+        return;
+      case common::kTypeDouble:
+        json_[name][key.data()] = opentelemetry::nostd::get<double>(value);
+        return;
+      case common::kTypeString:
+        json_[name][key.data()] = opentelemetry::nostd::get<std::string>(value).data();
+        return;
+      default:
+        return;
```

Comment:
log not-supported warning?

Reply:
I just keep the same behaviour as it dose before. Maybe we can create a separate issue to add warning for this exporter?
I don't use `ElasticsearchLogExporter` and I'm not sure whether should we add additional data

Reply:
Yes, we can do it separately.